Assembler.UnitTests/AssemblerTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Moq;
+using NUnit.Framework;
+
+namespace Assembler.UnitTests
+{
+    [TestFixture]
+    public class MessagesAssemblerTests
+    {
+        private Mock<IResolver<FrameType, IHandler<BaseFrame, BaseMessageInAssembly>>> _resolverMock;
+        private Mock<IConverter<BaseMessageInAssembly, BaseAssembledMessage>> _converterMock;
+        private Mock<ITimeBasedCache<BaseMessageInAssembly>> _cacheMock;
+        private Mock<IHandler<BaseFrame, BaseMessageInAssembly>> _firstHandlerMock;
+        private Mock<IHandler<BaseFrame, BaseMessageInAssembly>> _secondHandlerMock;
+        private Base.MessagesAssembler _assembler;
```

Comment:
Why not add `using Assembler.Base;` and remove the `Base` from here (and lines 42, 65)?

Reply:
I had some naming changes that required it, but fixed it now.

Reply:
Cool, just remove the `Base` wherever you do `new Base.FramesAssembler(...)`

Reply:
f

Reply:
👍